### ISSUE TITLE
Not able to render component using react-rails

### ISSUE THREAD

**Issue by kushthedude** [reactjs/react-rails#1069]
Help us help you! Have you looked for similar issues? Do you have reproduction steps? [Contributing Guide](CONTRIBUTING.md#reporting-bugs)

### Steps to reproduce
- I added the `react-rails` gem in my gemfile, & ran 
```
$ bundle install
$ rails webpacker:install
$ rails webpacker:install:react
$ rails generate react:install
``` 
- I generated the HelloWorld component `rails g react:component HelloWorld greeting:string`
- I wrote the component in my view file `index/index`  with this `<%= react_component("HelloWorld", { greeting: "Hello from react-rails." }) %>`

### Expected behaviour

Greeting: Hello from react-rails should render on the page.

### Actual behaviour

The following doesn't render

### System configuration
**Sprockets or Webpacker version**:
**React-Rails version**: 2.6.1
**React_UJS version**:
**Rails version**: 6.0.3
**Ruby version**: 2.5.7

-------

(Describe your issue here)
**index.html.erb**
```html

<%= react_component("HelloWorld", { greeting: "Hello from react-rails." }) %>
```

**index_controller.rb**
```rb
class IndexController < ApplicationController
end
```

**app/javascript/packs/application.js**
```js
// This file is automatically compiled by Webpack, along with any other files
// present in this directory. You're encouraged to place your actual application logic in
// a relevant structure within app/javascript and only use these pack files to reference
// that code so it'll be compiled.

require("@rails/ujs").start()
require("turbolinks").start()
require("@rails/activestorage").start()
require("channels")


// Uncomment to copy all static images under ../images to the output folder and reference
// them with the image_pack_tag helper in views (e.g <%= image_pack_tag 'rails.png' %>)
// or the `imagePath` JavaScript helper below.
//
// const images = require.context('../images', true)
// const imagePath = (name) => images(name, true)
// Support component names relative to this directory:
var componentRequireContext = require.context("components", true);
var ReactRailsUJS = require("react_ujs");
ReactRailsUJS.useContext(componentRequireContext);
```

**Comment by kushthedude** [comment]
@BookOfGreg Any help will be appreciated!

**Comment by muchaya** [comment]
I had a similar problem although it was on an old rails app I updated to 6.0.3.1 recently. When you inspect the element you will likely see data-react-props but it will still not render anything. 

Have you included the javascript_pack_tag in **application.html.erb?**

  ` <%= javascript_pack_tag 'application' ,'data-turbolinks-track': 'reload' %>`

If you have it, add the following line in **javascript/packs/application.js** 

`console.log('Hello Webpacker is saying black lives matter')`

This should print to the console when you inspect the page that contains the react_component in your erb views.

If not, then go to the homepage of the webpacker gem to follow a few steps but I basically ran
**yarn upgrade and yarn install**. After that, I added the following files to **javascript/packs/application.js** 
 
`import 'core-js/stable'
import 'regenerator-runtime/runtime'`

I the re-installed react-js by following all the initial steps you took and voila ...
